Libplanet/Tx/Transaction.cs

```diff
@@ -107,7 +107,7 @@ internal Transaction(RawTransaction rawTx)
             : this(
                 rawTx.Nonce,
                 new Address(rawTx.Signer),
-                new PublicKey(rawTx.PublicKey),
+                new PublicKey(rawTx.PublicKey.ToArray()),  // FIXME: propagate?
```

Comment:
Does `FIXME: propagate?` means that `PublicKey`'s constructor also should take `ImutableArray<byte>`?

Reply:
Yes, I thought that `PublicKey`'s constructor also should take `ImmutableArray<byte>`, but this PR doesn't cover this. So I'll remove this comment. (And others too.)